playbooks/installer/ospd/main.yml

```diff
@@ -19,9 +19,17 @@
   hosts: undercloud
   gather_facts: no
   become: yes
-  become_user: "{{ installer.user.name }}"
+  become_method: sudo
   roles:
-      - {role: installer/ospd/images}
+      - role: installer/ospd/images
+        task: "{{ installer.images.task }}"
+        work_path: "{%- if installer.images.task == 'import' %}{{ '/home/{{ installer.user.name }}' }}{% endif %}"
```

Comment:
Not sure. IIUC, else workpath="". I think what you need here is ternary operator:

```yaml
work_path: {{ (installer.images.task == 'import') | ternary( '~' |expanduser, default(omit) ) }}
```